.github/workflows/simple.yaml

```diff
@@ -0,0 +1,114 @@
+---
+name: Simple workflow
+
+on:
+  push:
+    branches:
+      - main
+      # TODO update this
+      - setup-workflows
+  # pull_request:
+  #   types:
+  #     - opened
+  #     - synchronize
+  #     - reopened
+  #     - ready_for_review
+  #   branches:
+  #     - main
+
+# Prerequisites to run workflows:
+# Actions -> I understand my workflows...
+# Settings -> Actions -> General -> Workflow permissions -> Read and write permissions
+
+# env:
+#   ENABLED_STEPS: |
+#     checkout_repository
+#     get_package_name
+#     build_r_package
+#     run_r_cmd_check
+#     upload_artifact
+
+# When asked to, please add the steps from the list above:
+env:
+  ENABLED_STEPS: |
+    checkout_...
```

Comment:
here why we do not have full names (for ex `checkout_repository` ?) - users would need to complete this ?

Reply:
Yes, that's the idea - basically this is the easiest way I could think of to let users interact with the code to enable particular steps, yet on the other hand not to get too discouraged with YAML indentation.

Reply:
I like the idea :)

Reply:
wouldn't be easier to uncomment than rewrite? how about this:
```
env:
  ENABLED_STEPS: |
    dummy
#     checkout_repository
#     setup_r
#     setup_dependencies
#     get_package_name
#     build_r_package
#     run_r_cmd_check
#     upload_artifact
```

Reply:
I thought it might be easier to add the missing steps names, because they won't have to deal with the indentation, but I guess this way is also fine 🙂